core/image/compress_test.go

```diff
@@ -111,8 +148,8 @@ func outputDifference(name string, refImage *image.Data, finalImage *image.Data)
 	outputPath := name + "-output.png"
 	errorPath := name + "-error.png"
 
-	astcErrorMargin := float32(0.0001)
-	if diff <= astcErrorMargin {
+	errorMargin := float32(0.01)
```

Comment:
Widening the error margin doesn't feel right. Could this be to the low value of "effort" that you used in ETC compression?

Reply:
Well, I think the higher the value, this may actually get worse because we are compressing more with a lossy compression although I am not experienced on the topic so I cannot say. 

And that value, does not matter. We are not exporting those images and we cannot change it(unless we modify the compression code and see if there are any bug that affecting this).

The only thing important in that test is "Are we able to compress?" As long as texture format is changed and we can decompress our image back without a crash or error, test should succeed. Any quality or other property of the compression is irrelevant. In the full feature, user will be able to change compression intensity so they can see the performance effect on that but for the test purposes, it is irrelevant.

Reply:
And for some reason I use texture and image interchangeably, please read every `texture` as `image`.

Reply:
> I think the higher the value, this may actually get worse

The etc2comp README says this, hence my comment:
```
-effort <amount>              number between 0 and 100 to specify the encoding quality 
                              (100 is the highest quality)
```

> And that value, does not matter. We are not exporting those images and we cannot change it(unless we modify the compression code and see if there are any bug that affecting this).

Oh ok, I see your point: it's not necessary to test compression quality since that's the responsibility of the compression library, which is a third party. Thanks for the clarification here.